index.html

```diff
@@ -0,0 +1,43 @@
+
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+   <meta charset="UTF-8" />
+    
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>サイト制作課題</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link　
+
+
+      href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@100..900&display=swap" 
+      rel="stylesheet"
+　  />

```

Comment:
```suggestion
    <link
      href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@400;700&display=swap"
      rel="stylesheet"
    />

```
linkタグの後に空白がある影響でgooglefontsがうまく機能していないようです。その修正をお願いします。